lang/candidate.go

```diff
@@ -42,6 +42,8 @@ type Candidate struct {
 	// TriggerSuggest allows server to instruct the client whether
 	// to reopen candidate suggestion popup after insertion
 	TriggerSuggest bool
+
+	ResolveHook *ResolveHook
```

Comment:
Similar to above - do you mind adding a brief comment to explain the purpose of that field?